.cspell.json

```diff
@@ -7,7 +8,7 @@
     "**/*.jar"
   ],
   "language": "en",
-  "version": "0.1",
+  "version": "0.2",
```

Comment:
Upgrade from the legacy cspell config. This upgrade simply involves changing the `version` to `"0.2"`. 
References:
https://github.com/streetsidesoftware/cspell/issues/1532#issue-971713483
https://cspell.org/docs/getting-started/